inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -975,22 +975,27 @@ uint32_t GNAPlugin::QueueInference(const InferenceEngine::BlobMap &inputs, Infer
     return idx;
 }
 
-bool GNAPlugin::Wait(uint32_t request_idx) {
-    return WaitFor(request_idx, MAX_TIMEOUT);
+bool GNAPlugin::Wait(uint32_t idx) {
```

Comment:
The argument name became inconsistent with WaitFor.

Reply:
By the way, is it used at all now?

Reply:
Yes Wait() is still used.
Will restore request_idx param name.